app/containers/MessageActions.js

```diff
@@ -159,10 +165,11 @@ class MessageActions extends React.Component {
 		}
 	}
 
-	isOwn = props => props.actionMessage.u && props.actionMessage.u._id === props.user.id;
+	isOwn = props => props.message.u && props.message.u._id === props.user.id;
 
 	isRoomReadOnly = () => {
 		const { room } = this.props;
+		// TODO: We need to get -> is ReadOnly && !isOwner
```

Comment:
You can use it: https://github.com/RocketChat/Rocket.Chat.ReactNative/blob/develop/app/utils/room.js#L9, probably it is better.